Shake/Main.lean

```diff
@@ -451,12 +453,32 @@ def main (args : List String) : IO UInt32 := do
       IO.println "There are out of date oleans. Run `lake build` or `lake exe cache get` first"
       IO.Process.exit 1
 
+  -- Determine default module(s) to run Shake on
```

Comment:
Minor: do we capitalise shake or not? (I would instinctively lower-case it.) Also optional.